### ISSUE TITLE
mod incompat with tubular/stockpile with barrels and tube connectors

### ISSUE THREAD

**Issue by SDUBZ** [vini2003/Pipes#3]
https://github.com/branchpanic/stockpile/issues/82

**Comment by vini2003** [comment]
Could you try to update to latest, v1.0.1?

This issue should not happen if the latest, proper implementation if inventory handling in Tubular is being used.

**Comment by vini2003** [comment]
Filter option is broken, Tubular is probably doing something very wrong right now. Wait for update.

**Comment by SDUBZ** [comment]
yeah it happend when the connectors are very close together also tubes
so im guessing if you had a cover system to stop cables connecting then it would fix most of the cable issues

**Comment by SDUBZ** [comment]
@vini2003 with 1.0.1 http://puu.sh/EdWkD/40d4f7ce28.txt
heres the log too http://puu.sh/EdWnt/35b68a7e16.log

**Comment by vini2003** [comment]
I'll be finished practically rewriting the mod today, after implementing a lot of new mechanics for pipes. I promise this shouldn't take much longer.

**Comment by SDUBZ** [comment]
thank you for your hard work and take your time make sure its at the mos stable :)
